sources/packages/backend/libs/test-utils/src/factories/sin-validation.ts

```diff
@@ -1,8 +1,8 @@
 import { SINValidation, Student } from "@sims/sims-db";
 import * as faker from "faker";
 
-export function createFakeSINValidation(relations: {
-  student: Student;
+export function createFakeSINValidation(relations?: {
```

Comment:
why optional?

Reply:
Without making the student optional for SINValidation, we cannot pass a SINValidation object to `saveFakeStudent(relations?: { user?: User; sinValidation?: SINValidation })` because the student of the SINValidation is going to be created inside the saveFakeStudent and SINValidation can be assigned and saved only after the student is persisted, otherwise typeorm is going to throw the cyclic dependency error.